src/features/accounts/pages/account-page.tsx

```diff
@@ -18,7 +18,9 @@ export const accountFailedToLoadMessage = 'Account failed to load'
 
 const transformError = (e: Error) => {
   if (is404(e)) {
-    return new Error(accountInvalidAddressMessage)
+    const error = new Error(accountInvalidAddressMessage) as StatusError
+    error.status = 404
+    return error
```

Comment:
I had to add this to each of the transformErrors in each of pages because it strips out an identification that the message is a 404. I used this info the error-page to display the custom error message

Reply:
Sorry maybe a little more context is useful here.

Your code is doing this:
```
const transformError = (e: Error) => {
  if (is404(e)) {
    const error = new Error(accountInvalidAddressMessage) as StatusError
    error.status = 404
    return error
  }
}
```

You could instead do this:
```
const transformError = (e: Error) => {
  if (is404(e)) {
    e.message = accountInvalidAddressMessage
    return e
  }
}
```

You'd be preserving the thrown error (including the status field) and only mutating the message.

Reply:
Oh, I see. yeah, this is much cleaner. Made the change.